<img align="right" width="100" height="100" src="https://github.com/bcfgothenburg/Images/blob/master/ssf.jpg">


# Bioinformatics Methods in Biomedical Research (RIF14-0081)
*Projects funded by SSF (2016-2020) at the [Bioinformatics Core Facility](https://www.gu.se/en/core-facilities/bioinformatics-bcf), Sahlgrenska Academy, University of Gothenburg*

Our method development projects are divided in four categories:

**[Research projects](research-projects)**

We usually collaborate with researchers in the analysis of their data. Within these projects we were able to extend our in-depth participation without any cost to the researcher. We focused on projects with clinical and biological impact, and in need of tailored analysis, integration and visualization.

**[Tool development](tool-development)**: 

These projects emerged from analyses that we identified to be of general interest. We were able to devote time to implement workflows and tools. We are thankful to our collaborators who helped us to validate the results from the resulting tools.

**[Tool evaluation](tool-evaluation)**: 

It is common that within a specific analysis, several tools are available and, sometimes the lack of golden standards makes it difficult to select the best tool if it has not been previously evaluated. These projects show some of our experiences while evaluating these tools. 
    
**[Education](education)**: 

Besides aiding researchers in the analysis of their data we are also invested in teaching. Several of our courses were initially developed thanks to this grant. Now they are part of our standard [courses](https://www.gu.se/en/core-facilities/bioinformatics-bcf/education-and-training). Moreover, we were able to share our experience in adopting short bioinformatics courses for PhD during the pandemic.


<a name="research_projects"></a>
## Research projects

### Bioinformatics from an Engineering point of view
BCF contibutor(s); Marcela Dávila. In collaboration with Francisco Gomes, Jennifer Horkoff and Alexander Schliep, Chalmers University of Technology

   * Received further funding from [ICT SEED](https://www.chalmers.se/en/areas-of-advance/ict/news/Pages/The-granted-ICT-SEED-proposals-2018.aspx)
   * [Poster](https://github.com/bcfgothenburg/SSF/blob/master/2019_poster_obires.pdf)
   * [Heckmann Barbalho de Figueroa L. et al. (2019) A Modeling Approach for Bioinformatics Workflows. In: Gordijn J., Guédria W., Proper H. (eds) The Practice of Enterprise Modeling. PoEM 2019. Lecture Notes in Business Information Processing, vol 369. Springer, Cham](https://link.springer.com/chapter/10.1007%2F978-3-030-35151-9_11)
   * [Empirical study on software and process quality in bioinformatics tools
Katalin Ferenc, Konrad Otto, Francisco Gomes de Oliveira Neto, Marcela Dávila López, Jennifer Horkoff, Alexander Schliep
bioRxiv 2022.03.10.483804; doi: https://doi.org/10.1101/2022.03.10.483804](https://www.biorxiv.org/content/10.1101/2022.03.10.483804v1?rss=1)

### Distsribution of the RNAse MRP
BCF contributor(s): Marcela Dávila. In collaboration with Tore Samuelsson and Magnus Alm Rosenblad (UGoT)

   * [Magnus Alm Rosenblad, Marcela Dávila López & Tore Samuelsson (2021) The enigmatic RNase MRP of kinetoplastids, RNA Biology, 18:sup1, 139-147, DOI: 10.1080/15476286.2021.1952758](https://www.ncbi.nlm.nih.gov/pubmed/34308760)


### Gall bladder cancer profiling in hepatobiliar cell lines
BCF contributor(s): Sanna Abrahamsson and Marcela Dávila. In collaboration with Justo Bermejo (Heidelberg University)

   * [García P, Bizama C, Rosa L, et al. Functional and genomic characterization of three novel cell lines derived from a metastatic gallbladder cancer tumor. Biol Res. 2020;53(1):13. Published 2020 Apr 15. doi:10.1186/s40659-020-00282-7](https://biolres.biomedcentral.com/articles/10.1186/s40659-020-00282-7)
   * [Scherer, D.; Dávila López, M.; Goeppert, B.; Abrahamsson, S.; González Silos, R.; Nova, I.; Marcelain, K.; Roa, J.C.; Ibberson, D.; Umu, S.U.; Rounge, T.B.; Roessler, S.; Lorenzo Bermejo, J. RNA Sequencing of Hepatobiliary Cancer Cell Lines: Data and Applications to Mutational and Transcriptomic Profiling. Cancers 2020, 12, 2510. https://doi.org/10.3390/cancers12092510 ](https://www.mdpi.com/2072-6694/12/9/2510)

### Integration of omic data in bipolar disorder
BCF contributor(s): Katarina Truvé. In collaboration with Keiko Funa (UGoT)

   * [Poster](https://github.com/bcfgothenburg/SSF/blob/master/Poster_BD_SBW_2019_KT.pdf)
   * [Truvé, K., Parris, T.Z., Vizlin-Hodzic, D. et al. Identification of candidate genetic variants and altered protein expression in neural stem and mature neural cells support altered microtubule function to be an essential component in bipolar disorder. Transl Psychiatry 10, 390 (2020). https://doi.org/10.1038/s41398-020-01056-1](https://www.nature.com/articles/s41398-020-01056-1)

### Molecular regulation of homeostasis in the human liver
BCF contributor(s): Marcela Dávila. In collaboration with Christina Jern (UGoT)

   * [Olsson Lindvall M, Hansson L, Klasson S, Davila Lopez M, Jern C, Stanne TM. Hemostatic Genes Exhibit a High Degree of Allele-Specific Regulation in Liver. Thromb Haemost. 2019;119(7):1072-1083. doi:10.1055/s-0039-1687879](https://www.thieme-connect.de/products/ejournals/abstract/10.1055/s-0039-1687879)
   * [Olsson Lindvall M, Davila Lopez M, Klasson S, et al. A Comprehensive Sequencing-Based Analysis of Allelic Methylation Patterns in Hemostatic Genes in Human Liver. Thromb Haemost. 2020;120(2):229-242. doi:10.1055/s-0039-3401824](https://www.thieme-connect.de/products/ejournals/abstract/10.1055/s-0039-3401824)
   * [Olsson Lindvall M, Angerfors A, Andersson B, Nilsson S, Davila Lopez M, Hansson L, Stanne TM, Jern C. Comparison of DNA Methylation Profiles of Hemostatic Genes between Liver Tissue and Peripheral Blood within Individuals. Thromb Haemost. 2020 Nov 17. doi: 10.1055/s-0040-1720980. Epub ahead of print. PMID: 33202445.](https://pubmed.ncbi.nlm.nih.gov/33202445/)

   
<a name="research_development"></a>
## Tool development

### Machine learning biomarker identification in Radiotherapy
BCF contributor(s): Björn Andersson, Peidi Liu and Erik Lorentzen. In collaboration with Britta Langen (UGoT)

   * [Poster](https://github.com/bcfgothenburg/SSF/blob/master/SBW_2019_gbg_21_22oct_Final.pdf)
   * _Manuscript in preparation_

### InVi: Integration and visualizationof genomic data
BCF contributor(s): Luciano Fernández and Marcela Dávila. In collaboration with Christina Jern (UGoT)

   * [Poster](https://github.com/bcfgothenburg/SSF/blob/master/2017_circos_presentation_LF_2.pdf)
   * [Fernandez-Ricaud, L. and M. Dávila López, InVi: Integration and visualization of genomic data. 2019, Bioinformatics Core Facility. doi:10.13140/RG.2.2.24486.80969](http://dx.doi.org/10.13140/RG.2.2.24486.80969)

### [IONISER](https://bitbucket.org/xgotda/qtioniser/src/master/): Searching for glycoproteins
BCF contributor(s): Dagmara Gotlib. In collaboration with the [Proteomics Core Facility](https://cf.gu.se/english/Proteomics)
   * [Poster](https://github.com/bcfgothenburg/SSF/blob/master/Ioniser_poster_2019.pdf)
   * [Gotlib, D. and M. Dávila López, Ioniser: Assisting the identification of existing glycostructures for a given peptide. 2019, Bioinformatics Core Facility. doi:10.13140/RG.2.2.34664.55049](http://dx.doi.org/10.13140/RG.2.2.34664.55049)

### mitoPD: Mitochondria-specific peak detection
BCF contributor(s): Sanna Abrahamsson and Marcela Dávila. In collaboration with Mara Domio and Sjöerd Wanrooij (Umeå University)

_Ongoing_

### [Odyssey 2.1.1](https://github.com/AlinaO311/Odyssey): Imputation of genomic data
BCF contributor(s): Björn Andersson and Alina Orozco. In collaboration with Tara Stanne (UGoT)

  * [Orozco, A., Odyssey 2.1.1: Imputation of genomic data. 2022, Bioinformatics Core Facility.](https://www.gu.se/sites/default/files/2022-04/Odyssey_2022.pdf)

### [PΨFinder](https://github.com/SannaAb/Pseudogenes): Novel processed pseudogenes detection tool
BCF contributor(s): Sanna Abrahamsson and Marcela Dávila. In collaboration with Anna Rohlin (Laboratory Medicine at UGoT)

   * [Poster](https://github.com/bcfgothenburg/SSF/blob/master/2017_poster_Pseudogenes_ForPrint.pdf)
   * [Abrahamsson, S., Eiengård, F., Rohlin, A. et al. PΨFinder: a practical tool for the identification and visualization of novel pseudogenes in DNA sequencing data. BMC Bioinformatics 23, 59 (2022). https://doi.org/10.1186/s12859-022-04583-4](https://www.ncbi.nlm.nih.gov/pubmed/35114952)

### REAPER: A light-weight file monitor
BCF contributor(s): Dagmara Gotlib. In collaboration with the [Proteomics Core Facility](https://cf.gu.se/english/Proteomics)
   * [Gotlib, D., Reaper: A lightweight file monitor. 2020, Bioinformatics Core Facility.](https://www.gu.se/sites/default/files/2022-02/Reaper_2020.pdf)

[Proteomics Core Facility]

### [TC-Hunter](https://github.com/vborjesson/TC_hunter): Identification of transgenic insertion sites within the host genome
BCF contributor(s): Vanja Börjesson and Marcela Dávila. In collaboration with Jelena Milosevic (Karolinska Institutet)

   * [Poster](https://github.com/bcfgothenburg/SSF/blob/master/2018_0828_TChunter.pdf)
   * [Börjesson, V., Martinez-Monleon, A., Fransson, S. et al. TC-hunter: identification of the insertion site of a transgenic gene within the host genome. BMC Genomics 23, 149 (2022). https://doi.org/10.1186/s12864-022-08376-0](https://www.ncbi.nlm.nih.gov/pubmed/35184734)


<a name="tool-evaluation"></a>
## Tool evaluation

### MaSILiM: Mass Spectometry Imaging and Light microscopy fusion
BCF contributor(s): Dagmara Gotlib. In collaboration with the Centre for [Cellular Imaging](https://cf.gu.se/english/centre_for_cellular_imaging)

   * [Gotlib, D., MaSIliM: Mass Spectrometry Imaging and Light Microscopy fusion 2021, Bioinformatics Core Facility. doi:10.13140/RG.2.2.15056.17920](http://dx.doi.org/10.13140/RG.2.2.15056.17920)
   
### Mutational signatures
BCF contributor(s): Katarina Truvé. In collaboration with Anna Rohlin (Laboratory Medicine at UGoT)

   * [Truve, K., Comparisons of two tools for identifying mutational signatures in cancer. 2022, Bioinformatics Core Facility.](https://www.gu.se/sites/default/files/2022-03/Signatures_2022.pdf)

### Pathway Analysis software evaluation
BCF contributor(s): Annelie Angerfors

   * [Angerfors, A. and M. Davila Lopez, Pathway analysis software evaluation. 2021, Bioinformatics Core Facilty. doi:10.13140/RG.2.2.27650.56005.](http://dx.doi.org/10.13140/RG.2.2.27650.56005)

### Pseudotime analysis tools for snRNAseq data
BCF contributor(s): Vanja Börjesson. In collaboration with Malin Johansson (UGoT)

   * [Poster](https://github.com/bcfgothenburg/SSF/blob/master/20191007_pseudotime.pdf)
   * [Börjesson, V., Comparison of bioinformatic tools for pseudo time analysis. 2020, Bioinformatics Core Facility.](https://www.gu.se/sites/default/files/2022-02/pseudotimeTools_2020.pdf)

<a name="education"></a>
## Education


### Online education during the COVID-19 pandemic
BCF contributor(s): Marcela Davila and Sanna Abrahamsson

   * [Sanna Abrahamsson, Marcela Dávila López, Comparison of online learning designs during the COVID-19 pandemic within bioinformatics courses in higher education, Bioinformatics, Volume 37, Issue Supplement_1, July 2021, Pages i9–i15, doi:10.1093/bioinformatics/btab304](https://www.ncbi.nlm.nih.gov/pubmed/34252967)

### Proteomics data analysis workflow
BCF contributor(s): Jari Martikainen. In collaboration with [Proteomics Core Facility](https://cf.gu.se/english/Proteomics)

   * [Martikainen, J. and M. Davila Lopez, Proteomics data analysis: design, missing values and batch effect. 2021, Bioinformatics Core Facility. doi:10.13140/RG.2.2.24486.80969](http://dx.doi.org/10.13140/RG.2.2.22816.46089)

 
***
 
**We are thankful to all collaborators and students that participated in helping us to advance bioinformatics and statistical aspects within biomedical research**  

[Bioinformatics Core Facility](https://www.gu.se/en/core-facilities/bioinformatics-bcf)
